vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/DatabaseBackedTestConfiguration.java

```diff
@@ -17,7 +17,7 @@ public class DatabaseBackedTestConfiguration {
   private static final TemporaryFolder unloadDirectory = new TemporaryFolder();
 
   public static JdbcDatabaseContainer getTestDatabaseContainer() {
-    return new PostgreSQLContainer("postgres:13-alpine")
```

Comment:
Why the downgrade?

Reply:
We use Postgres 12 in all the other tests and in Vidarr deploy -  downgrading of this one I think was just missed.

Reply:
The changelog indicates `Update database integration tests to use same postgresql version as other ITs and Vidarr deploys`, which makes sense

Reply:
(deleted my comment because I hadn't seen that Mike was more eloquent about the same thing)